backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt

```diff
@@ -14,7 +14,7 @@ class DatabaseActionTrackerManagerTest : TestContainerProvider() {
   private val manager = DatabaseActionTrackerManager(database, EmptyCoroutineContext)
 
   @Test fun `email action entries are recorded in both tables`() = suspendTest {
-    driver.insertIssue(1, "this is an issue", "tweet", "url.com")
+    val id = driver.insertIssue("this is an issue", "tweet", "url.com")
     driver.insertMemberOfCongress(DEFAULT_MEMBER_OF_CONGRESS.copy(bioguideId = "hello"))
     manager.trackActionSendEmail("foo@foo.com", "hello", 1)
```

Comment:
Do we need to update the last argument here?

```suggestion
    manager.trackActionSendEmail("foo@foo.com", "hello", id)
```

Same comment applies to some of the calls below.

Reply:
yes, good call.